src/Video/VideoErrorCheck.cpp

```diff
@@ -0,0 +1,27 @@
+#include "VideoErrorCheck.hpp"
+
+#include <GL/glew.h>
+#include <Utility/Log.hpp>
+
+VideoErrorCheck::VideoErrorCheck(const std::string& name) {
+    this->name = name;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'name' is assigned in constructor body. Consider performing initialization in initialization list. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuseInitializationList)